cybergis_compute_client/CyberGISCompute.py

```diff
@@ -18,6 +18,12 @@
 from IPython.display import display, Markdown, Javascript
 
 
+class ParamAccumulator:
```

Comment:
Not quite understanding what the role of this helper function is at this point. It seems to be a wrapper around a dictionary that doesn't provide any additional functionality.

Reply:
Yeah the role for that class is to handle all of the input preprocessing that might be needed. For e.g. json paths ( working on it rn ) or some other input format that we might wanna support.